.github/libs/GitUtils.ts

```diff
@@ -192,19 +230,26 @@ function getValidMergedPRs(commits: CommitType[]): number[] {
  * Takes in two git tags and returns a list of PR numbers of all PRs merged between those two tags
  */
 async function getPullRequestsMergedBetween(fromTag: string, toTag: string) {
-    console.log(`Looking for commits made between ${fromTag} and ${toTag}...`);
-    const commitList = await getCommitHistoryAsJSON(fromTag, toTag);
-    console.log(`Commits made between ${fromTag} and ${toTag}:`, commitList);
-
-    // Find which commit messages correspond to merged PR's
-    const pullRequestNumbers = getValidMergedPRs(commitList).sort((a, b) => a - b);
-    console.log(`List of pull requests merged between ${fromTag} and ${toTag}`, pullRequestNumbers);
-    return pullRequestNumbers;
+    console.log(`Looking for commits made between ${fromTag} and ${toTag}`);
+
+    const apiCommitList = await getCommitHistoryBetweenTags(fromTag, toTag);
+    const apiPullRequestNumbers = getValidMergedPRs(apiCommitList).sort((a, b) => a - b);
+    console.log(`[API] Found ${apiCommitList.length} commits.`);
+    console.error(`[API] Parsed PRs: ${apiPullRequestNumbers.join(', ')}`);
+
+    // eslint-disable-next-line deprecation/deprecation
+    const gitCommitList = await getCommitHistoryAsJSON(fromTag, toTag);
+    const gitLogPullRequestNumbers = getValidMergedPRs(gitCommitList).sort((a, b) => a - b);
+    console.log(`[git log] Found ${gitCommitList.length} commits.`);
+    console.error(`[git log] Parsed PRs: ${gitLogPullRequestNumbers.join(', ')}`);
```

Comment:
after importing `core` from `@actions/core`:

```suggestion
core.startGroup('Parsed PRs:');
core.info(JSON.stringify(gitLogPullRequestNumbers));
core.endGroup();
```

That will make this potentially very large log line collapsible so we can include it for debugging without the overall logs hard to read.